Gemfile

```diff
@@ -2,4 +2,6 @@ source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
 gem 'fastimage'
 gem 'img_checker'
+#gem 'jekyll-sitemap'
```

Comment:
Remove comments as well

Reply:
Also, link the installation file to readme

Reply:
@vaibhavsingh97 these comments needs to be uncommented to work on windows, referred on Installation Steps.

Reply:
@rupav PLease make seperate readme file for windows/Linux

Reply:
I made them commented because it wasn't earlier added (and thus may create problem in other OS setup).
In next task, (setting up in Linux or mac) I or other mentor can add docs/installation folder and then we can add installation_windows.md file in it. just like @ParthS007 did in badgeyay. Is it fine?

Reply:
ok, I will make that, thanks for working on it